app/components/channel_drawer/channel_drawer.js

```diff
@@ -45,15 +48,25 @@ export default class ChannelDrawer extends PureComponent {
         theme: PropTypes.object.isRequired
     };
 
-    state = {
-        openDrawer: false,
-        openDrawerOffset: DRAWER_INITIAL_OFFSET
-    };
-
     swiperIndex = 1;
 
+    constructor(props) {
+        super(props);
+
+        const orientation = Orientation.getInitialOrientation();
```

Comment:
Should we also move orientation into the redux store?

Reply:
Thought about it, and I guess I've missed this as well as to identify if the device is tablet or not. Maybe that can be a further improvement?